peer.go

```diff
@@ -438,7 +438,12 @@ func (p *peer) loadActiveChannels(chans []*channeldb.OpenChannel) error {
 
 		// Skip adding any permanently irreconcilable channels to the
 		// htlcswitch.
-		if dbChan.ChanStatus() != channeldb.ChanStatusDefault {
+		switch {
```

Comment:
Only the first bit is always default or none. For the rest, they're treated as a bit vector, so we still need to do a bitmask to compare. Additionally, it's nice that this is explicit, rather than a blanket check since it'll force us to consider any future combinations.